atat_internal_api.yaml

```diff
@@ -349,7 +349,7 @@ paths:
       x-amazon-apigateway-integration:
```

Comment:
I know there are no `404` errors in the spec for `POST createApplication`, but if the `portfolioId` is incorrect we get a `404` error.  Should that be added (around lines 348)?

Reply:
Don't want to edit the spec, but when we do eventually fix `allOf` we should include those fixes.